cpp/src/column/column_view.cpp

```diff
@@ -55,8 +56,7 @@ void prefetch_col_data(ColumnView& col, void const* data_ptr, std::string_view k
         scv.chars_size(cudf::get_default_stream()) * sizeof(char),
         cudf::get_default_stream());
     } else {
-      std::cout << key << ": Unsupported type: " << static_cast<int32_t>(col.type().id())
-                << std::endl;
+      CUDF_LOG_DEBUG("Unsupported type: %d", static_cast<int32_t>(col.type().id()));
```

Comment:
Just a note that this will no be compiled by default. If you want an option to log this in our regular builds, INFO can be used instead (if this does not happen often).

Reply:
Yes, that's totally fine. We probably almost-never need this level of info.

Reply:
Yes, debug was an intentional choice here. I want this print compiled out by default.

Reply:
> (if this does not happen often).

This is currently happening any time that nested types are used with cudf.pandas. I'm surprised that nobody has noticed or complained about the verbosity before 😅